Documentation/admin.rst

```diff
@@ -155,8 +155,10 @@ Here is an example Daemon Set definition:
 	        args:
 	          - "-t"
 	          - "vxlan"
-	          - "--etcd-config-path"
-	          - "/var/lib/cilium/etcd-config.yml"
+	          - "--kvstore"
```

Comment:
I can't find where `with-cni.sh` is defined so I don't actually know if this is right..